internal/ipc/internal/flisten/flisten.go

```diff
@@ -0,0 +1,60 @@
+package flisten
```

Comment:
`flisten` is not self-explanatory. Please either rename or add some inline documentation explaining what its responsibilities are.